misc/bazel/internal/install.py

```diff
@@ -42,7 +44,22 @@
 script = runfiles.Rlocation(opts.pkg_install_script)
 
 if destdir.exists() and opts.cleanup:
-    shutil.rmtree(destdir)
+    if platform.system() == 'Windows':
+        # On Windows we might have virus scanner still looking at the path so
+        # attempt removal a couple of times sleeping between each attempt.
+        for retry_delay in [1, 2, 2]:
+            try:
+                shutil.rmtree(destdir)
+                break
+            except OSError as e:
+                if e.winerror == 32:
+                    time.sleep(retry_delay)
+                else:
+                    raise
+        else:
+            shutil.rmtree(destdir)
+    else:
+        shutil.rmtree(destdir)
```

Comment:
maybe to lower the ugliness we can extract our own `rmtree`, replace `break` with return, and collapse these two `else`s into one fall-through 
```suggestion
    shutil.rmtree(destdir)
```
?